viewer/packages/utilities/src/three/ThickLine.ts

```diff
@@ -61,77 +70,85 @@ export class MeasurementLine {
   }
 
   /**
-   * Update the measuring line end point.
-   * @param endPoint Second point of the line to end the measurement.
+   * Update the line end point.
+   * @param endPoint Second point of the line
    */
-  updateLine(endPoint: THREE.Vector3): void {
-    this._endpoint.copy(endPoint);
+  setLineEndPoint(endPoint: THREE.Vector3): void {
+    this._endPosition.copy(endPoint);
     this._meshes.position.copy(this.getMidPointOnLine());
     this._meshes.lookAt(endPoint);
-    this._meshes.scale.set(0, 0, endPoint.distanceTo(this._startPos));
+    this._meshes.scale.set(0, 0, endPoint.distanceTo(this._startPosition));
   }
 
   /**
-   * Updates the measuring line clipping planes
+   * Updates the line clipping planes
    * @param clippingPlanes current active global clipping planes.
    */
-  updateLineClippingPlanes(clippingPlanes: THREE.Plane[]): void {
+  setLineClippingPlanes(clippingPlanes: THREE.Plane[]): void {
     const visible =
-      !this.clippingPlanesContainPoint(clippingPlanes, this._startPos) ||
-      !this.clippingPlanesContainPoint(clippingPlanes, this._endpoint);
+      !this.clippingPlanesContainPoint(clippingPlanes, this._startPosition) ||
+      !this.clippingPlanesContainPoint(clippingPlanes, this._endPosition);
     this._meshes.visible = !visible;
   }
 
   /**
-   * Get the distance between the measuring line start point & end point.
+   * Get the distance between the line start point & end point.
    * @returns Return distance between start & end point of the line.
    */
-  getMeasuredDistance(): number {
-    return this._endpoint.distanceTo(this._startPos);
+  getLineLength(): number {
+    return this._endPosition.distanceTo(this._startPosition);
   }
 
   /**
    * Calculate mid point on the Line.
    * @returns Returns mid point between start and end points.
    */
   getMidPointOnLine(): THREE.Vector3 {
-    return this._endpoint.clone().add(this._startPos).multiplyScalar(0.5);
+    return this._endPosition.clone().add(this._startPosition).multiplyScalar(0.5);
   }
 
   /**
    * Update current line width.
-   * @param lineWidth Width of the measuring line mesh.
+   * @param lineWidth Width of the line mesh.
    */
-  updateLineWidth(lineWidth: number): void {
+  setLineWidth(lineWidth: number): void {
     this._adaptiveWidthLineMaterial.linewidth = lineWidth;
   }
 
   /**
    * Update current line color.
-   * @param color Color of the measuring line mesh.
+   * @param color Color of the line mesh.
    */
-  updateLineColor(color: THREE.Color): void {
+  setLineColor(color: THREE.Color): void {
     this._fixedWidthLineMaterial.color = this._adaptiveWidthLineMaterial.color = color;
   }
 
+  /**
+   * Set visibility
+   */
+  setVisibility(visible: boolean): void {
+    this._fixedWidthLineMaterial.visible = this._adaptiveWidthLineMaterial.visible = visible;
+  }
+
   /**
    * Generate line geometry and create the mesh.
    */
-  private startLine(): void {
+  private initializeMeshes(): void {
```

Comment:
Maybe better to be named `initializeLineMeshes`, so that it will look similar to other methods names.

Reply:
Refactoring class entirely